### DISCUSSION TITLE
[Request]: New Name!

### DISCUSSION THREAD

**Discussion by AtomikkuSan** [zer0Kerbal/SpaceShuttleEngines#11] [General]
### Description of the new feature / enhancement

When I was first developing this mod, I originally named it Shuttle Engines Rebuilt and I'd like to name this continuation with something similar. I'd want to rename it to Space Shuttle Engines Reflown.

### Proposed technical implementation details

Change the Repo name.

**Comment by zer0Kerbal** [comment]
we should leave the name as is - better title/brand recognition, that is if we get the permission from the original copyright holder - 

### will ask @Dtobi later today.

@AtomikkuSan, repo is not ready yet. Have massive amount to do before anything is done to it. Will finish it once I finish prepping GA

**Comment by AtomikkuSan** [comment]
Wouldn't @sarbian be a better person to ask?

**Comment by zer0Kerbal** [comment]
> Wouldn't @sarbian be a better person to ask?

he has already consented, but according to the OP in the KSP forums, and his thread - @Dtobi requires asking him, the original and only copyright/trademark holder (because of the CC _**ND**_ clause of the license).

That will happen later today. 

![image](https://user-images.githubusercontent.com/39887717/161827344-a2f5e5ef-01d8-40cc-a7f8-7b135bee15d0.png)

**Comment by AtomikkuSan** [comment]
Okay, thanks for the clarification!

I really hate the ND clause, it makes mod developing a lot harder, especially on the legal front.

**Reply by zer0Kerbal** [reply]
@AtomikkuSan  it does pose certain _unique_ challenges, but the CC ND license does a decent job of protecting the author's hard work and effort.

**Comment by zer0Kerbal** [comment]
> Okay, thanks for the clarification!
> 
> I really hate the ND clause, it makes mod developing a lot harder, especially on the legal front.

respecting the original author's wishes, and just general respect in general. 

yes, it can be a PITA, and many Many MANY mods have died because of this, but at the same time it has kept several key mods alive.

experience has taught me to go very carefully with licenses and make sure get the full permission. which we will get. :D

**Comment by AtomikkuSan** [comment]
> yes, it can be a PITA, and many Many MANY mods have died because of this, but at the same time it has kept several key mods alive.

Which ones stayed alive?



> experience has taught me to go very carefully with licenses and make sure get the full permission. which we will get. :D

I can't wait!

**Comment by dtobi** [comment]
Hi guys. What's up?

**Comment by zer0Kerbal** [comment]
> Hi guys. What's up?

Welcome @dtobi ! Just about to shoot an dm to you **_et al_** on the KSP forums concerning adopting. With a CC ND 3.0 license must dot i's, cross t's, and zip the zeds.

**Comment by zer0Kerbal** [comment]
@AtomikkuSan just want to verify your forum name so I can include you in the adoption letter. AtomicTech correct?

**Comment by zer0Kerbal** [comment]
@ dtobi - KSP Forum Direct Message sent. Also has my email address.

@AtomikkuSan I don't expect you to want to help with all of them - but I suspect you will want to. :D

I think I got all of them, all except:

* SmartParts (CC-BY-NC-SA-3.0) was adopted by linuxgurugamer
* CoolRockets is by Sarbian now, am sure he wants to keep it. 👍

**Comment by AtomikkuSan** [comment]
OH YES, IT'S (MOD DEVELOPING) SHOWTIME!

**Reply by zer0Kerbal** [reply]
@AtomikkuSan just kindly wait until I say the repo(s) are ready - and you finally understand (I hope) the benefits of this - just look at how much waiting helped with that mega-mod you adopted. Same thing here.

**Comment by AtomikkuSan** [comment]
> @AtomikkuSan just want to verify your forum name so I can include you in the adoption letter. AtomicTech correct?

Yep!

**Comment by AtomikkuSan** [comment]
@dtobi can we rename the project to Space Shuttle Engines Reflown and, more or less, rename every part with a more technical sounding names and better descriptions?

**Comment by AtomikkuSan** [comment]
> @AtomikkuSan I don't expect you to want to help with all of them - but I suspect you will want to. :D

So, which ones do we have?

**Comment by dtobi** [comment]
> @dtobi can we rename the project to Space Shuttle Engines Reflown and, more or less, rename every part with a more technical sounding names and better descriptions?

Would Klockheed Martian stay as brand name? In general, I’m fine with Space Shuttle Engines Reflown. It would be nice if you would still credit me as original author, though. 

Otherwise you’re free to rename parts, change code or add new models, of course.

**Comment by zer0Kerbal** [comment]
> Klockheed Martian

@dtobi I personally prefer keeping the brand intact - so keeping the wonderful pun Klockheed Martian as manufacturer, agency, and total brand. Changing the name of the add-on is not my first choice because it is disrespectful to you and all the hard graft you put into it in the first place. Changing the name also sends players the wrong message.

---

@dtobi legally can't until we have an answer to the forum's direct message since the license is CC ND. This is because the original source of the files and license comes from the forums and not GitHub. 

The permission must come from the original source.

Just doing the legal due diligence.

Mahalo nui loa!

**Comment by AtomikkuSan** [comment]
> > @dtobi can we rename the project to Space Shuttle Engines Reflown and, more or less, rename every part with a more technical sounding names and better descriptions?
> 
> Would Klockheed Martian stay as brand name? In general, I’m fine with Space Shuttle Engines Reflown. It would be nice if you would still credit me as original author, though.
> 
> Otherwise you’re free to rename parts, change code or add new models, of course.

I'm okay with all the current parts staying under Klockheed Martian, but can I change any of the new parts to be under AtomicTech Inc.?

**Comment by AtomikkuSan** [comment]
@zer0Kerbal, I've updated all the new pull requests to have Klockheed Martian as their manufacterer.

**Reply by zer0Kerbal** [reply]
@AtomikkuSan thank you. I will get back to this repo shortly.

Today should be KAMP and setting up another repo like the one I did yesterday.

I wonder if it is supposed to be:

1. Klockheed Martian 
2. KlockheedMartian
3. Klockheed-Martian

@dtobi which would you prefer?

and the tagline "Your Mission is Kerbal 々" 📟 
and the Japanese symbol 々is the logo.

**Comment by zer0Kerbal** [comment]
> new parts

👍 

just waiting on @dtobi to reply to the forum direct message; it's a sticky pickle this conundrum we find ourselves facing, need a pickle jar opener
https://www.youtube.com/watch?v=2VIWdRg__PM

**Comment by zer0Kerbal** [comment]
nearly in business now. 

converting this to a discussion. 

will keep everyone informed also will update [here](https://github.com/zer0Kerbal/SpaceShuttleEngines/discussions/4) as the repo is prepared, hopefully my schedule will allow me to prioritize this wonderous parts add-on.

**Comment by dtobi** [comment]
Its a pun based on Lockheed Martin so Klockheed Martian would bet the way to go, in my opinion.

Dtobi

**Comment by AtomikkuSan** [comment]
Any updates on this side of the repo?

**Reply by zer0Kerbal** [reply]
waiting for the public statement.

**Reply by AtomikkuSan** [reply]
Gotcha!

**Reply by zer0Kerbal** [reply]
@AtomikkuSan meanwhile CurseForge has cleared the last hurdle for two other mods so am working on them after AoR.

**Reply by zer0Kerbal** [reply]
@AtomikkuSan _soon™er™ish™_

**Comment by dtobi** [comment]
Are you waiting for me? I dropped the ball? What do I need to do?

Tobias

**Reply by zer0Kerbal** [reply]
@dtobi 
you asked the same thing in the direct message on the KSP forums. 

Please read the reply.

**Reply by zer0Kerbal** [reply]
@dtobi may I email you at dertobi@a********l.de? You also have my email address from the direct message on the KSP forums.

**Comment by dtobi** [comment]
Sure. Go ahead.

Tobias

**Reply by zer0Kerbal** [reply]
@dtobi 📧 inbound. Thank you.

_email is MIME signed to verify it is me_

**Reply by zer0Kerbal** [reply]
Thank you [@dtobi](https://forum.kerbalspaceprogram.com/index.php?/profile/90882-dtobi/)!

**Comment by AtomikkuSan** [comment]
So, are we good to go?

**Reply by zer0Kerbal** [reply]
@AtomikkuSan actually no. As expected there has been a snag on the forums due to the license. The issue is being actively worked on by multiple parties, including the forum mods. news when there is news.

**Reply by AtomikkuSan** [reply]
Where's this event going down? On the forum thread for Space Shuttle Engines?

**Reply by zer0Kerbal** [reply]
its a direct message thread between the mods and myself (and @dtobi )
Been cleared up and now working on the verbiage/wording to satisfy the forum rules. Once the new wording license is posted we are clear to proceed.
